packages/dds/tree/src/feature-libraries/flex-tree/unboxed.ts

```diff
@@ -28,10 +29,14 @@ import { makeTree } from "./lazyNode.js";
  */
 export function unboxedTree<TSchema extends FlexTreeNodeSchema>(
 	context: Context,
-	schema: TSchema,
+	schema: TSchema | TSchema[],
```

Comment:
Having to transform the ReadonlySet<> we have in some places to an array is a bit of a perf hit; might be preferable to keep the commented code down below instead of trying to route the path with more than one possible type through unboxedTree().